pkg/host/internal/sriov/sriov.go

```diff
@@ -376,6 +376,11 @@ func (s *sriov) configureHWOptionsForSwitchdev(iface *sriovnetworkv1.Interface)
 		log.Log.Error(err, "configureHWOptionsForSwitchdev(): fail to read current flow steering mode for the device", "device", iface.PciAddress)
 		return err
 	}
+	if currentFlowSteeringMode == "" {
```

Comment:
alternative is to omit this check and try to set steering mode to smfs.

can you share more details on what is the behaviour and on which kernel ?

after setting the parameter via netlink are we able to query it back ?

Reply:
after internal discussion, it seems like a kernel bug so we would want to protect against it but not attempt to configure.